sites/public/pages/applications/review/confirmation.tsx

```diff
@@ -26,6 +28,20 @@ const ApplicationConfirmation = () => {
 
   const imageUrl = imageUrlFromListing(listing, parseInt(process.env.listingPhotoSize))
 
+  const reviewOrder = useMemo(() => {
+    if (listing) {
+      if (listing.reviewOrderType == ListingReviewOrder.lottery) {
```

Comment:
A listing can be saved as having a review order type of lottery and no application due date, so the existence of that field should also be checked so we can't end up in a broken string state.